src/meta/processors/job/BalanceJobExecutor.cpp

```diff
@@ -88,6 +88,36 @@ nebula::cpp2::ErrorCode BalanceJobExecutor::save(const std::string& k, const std
   return rc;
 }
 
+void BalanceJobExecutor::insertOneTask(
+    const BalanceTask& task, std::map<PartitionID, std::vector<BalanceTask>>* existTasks) {
+  std::vector<BalanceTask>& taskVec = existTasks->operator[](task.getPartId());
```

Comment:
Just curious, why not use reference instead?

Reply:
> Just curious, why not use reference instead?

just follow google style, an output param should be non-const pointer